### PR TITLE
add ja.json

### PR DESCRIPTION
**Pull request by deyuaru**

### CODE REVIEW

**Comment by KycKyc** [comment]
@deyuaru  Will you continue working on your translation?

File: locales/ja.json [hunk 1]
```diff
@@ -0,0 +1,932 @@
...
+    "app.auctions.modal.lich.ephemera.no": "No",
+
+    "app.auctions.modal.riven.label.attribute": "Attribute",
+    "app.auctions.modal.riven.label.effect": "Effect",
+    "app.auctions.modal.riven.label.value": "Value",
+    "app.auctions.modal.riven.label.weapon": "武器名前",
+    "app.auctions.modal.riven.label.modName": "リベンモッドランク",
+    "app.auctions.modal.riven.label.mastery": "マスタリー",
+    "app.auctions.modal.riven.label.modRank": "モッドランク",
+    "app.auctions.modal.riven.label.reRolls": "Re-rolls",
```
Comment (Stream5710) [inline locales/ja.json:450]: `"app.auctions.modal.riven.label.reRolls": "サイクル数",`

File: locales/ja.json [hunk 2]
```diff
@@ -0,0 +1,932 @@
...
+    "app.auctions.tips.auctionType.buyout": " - Auctions that have a buyout price.",
+    "app.auctions.tips.auctionType.sellOrder.title": "Sell order",
+    "app.auctions.tips.auctionType.sellOrder": " - Just a sale order, not an auction, no bids.{br}Yes, sell orders under auctions tab, it's a bit weird, but flexible.",
+
+    "app.auctions.alert.reminder": "It is recommended to {reminderLink}, in order to be always up to date with your auctions.",
+    "app.auctions.alert.reminder.link": "enable desktop notifications",
+
+    "app.auctions.tabs.mostPopular": "Popular",
+    "app.auctions.tabs.mostRecent": "Most recent",
+    "app.auctions.tabs.search": "サーチ",
```
Comment (Stream5710) [inline locales/ja.json:224]: `"app.auctions.tabs.search": "検索",`

File: locales/ja.json [hunk 3]
```diff
@@ -0,0 +1,932 @@
...
+    "app.user.status.ingame": "ゲーム内でオンライン",
+    "app.user.status.online": "オンライン",
+    "app.user.status.offline": "オフライン",
+    "app.user.status.invisible": "インビジブル",
+    "app.user.status.connecting": "接続しています...",
+    "app.user.status.lastSeen": "最後オンライン時は...",
+
+    "app.order.marker.wts": "wts",
+    "app.order.marker.sold": "売済み",
+    "app.order.marker.wts.long": "売れたい",
```
Comment (Stream5710) [inline locales/ja.json:103]: `"app.order.marker.wts.long": "売りたい",`

File: locales/ja.json [hunk 4]
```diff
@@ -0,0 +1,932 @@
...
+
+    "app.user.status.ingame": "ゲーム内でオンライン",
+    "app.user.status.online": "オンライン",
+    "app.user.status.offline": "オフライン",
+    "app.user.status.invisible": "インビジブル",
+    "app.user.status.connecting": "接続しています...",
+    "app.user.status.lastSeen": "最後オンライン時は...",
+
+    "app.order.marker.wts": "wts",
+    "app.order.marker.sold": "売済み",
```
Comment (Stream5710) [inline locales/ja.json:102]: `"app.order.marker.sold": "売却済",`

File: locales/ja.json [hunk 5]
```diff
@@ -0,0 +1,932 @@
...
+    "app.order.marker.wts.long": "売れたい",
+    "app.order.marker.wtb": "wtb",
+    "app.order.marker.bought": "買済み",
```
Comment (Stream5710) [inline locales/ja.json:105]: `"app.order.marker.bought": "購入済",`

File: locales/ja.json [hunk 6]
```diff
@@ -0,0 +1,932 @@
...
+    "app.auction.onPause.owner": "Please contact the user in the game and process the trade deal. If all goes well, you can mark the auction as \"completed\" (top right corner).{br}In case of failed transaction/scam/trolling, to continue your auction, you can un-mark or ban the winner. Also you can try to pick another winner.",
+    "app.auction.onPause.participant": "Do not leave the auction, the owner may reconsider or cancel his choice of the winner (auction may continue).",
+    "app.auction.completed": "The auction was marked as complete.",
+    "app.auction.completed.winnerIs": "The winner is: ",
+    "app.auction.bans.hint": "Bans applies only to this specific auction.",
+    "app.auction.bans.empty": "No one is banned, yet.",
+    "app.auction.bans.button.removeBan": "Unban",
+    "app.auction.clipboard.template": "/w {ingameName} Hi!",
+
+    "app.auction.riven.modRank": "モッドランク",
```
Comment (Stream5710) [inline locales/ja.json:423]: `"app.auction.riven.modRank": "Modランク",`

File: locales/ja.json [hunk 7]
```diff
@@ -0,0 +1,932 @@
...
+    "app.auction.riven.mastery": "マスタリー",
+    "app.auction.riven.reRolls": "Re-rolls",
```
Comment (Stream5710) [inline locales/ja.json:425]: `"app.auction.riven.reRolls": "サイクル数",`

File: locales/ja.json [hunk 8]
```diff
@@ -0,0 +1,932 @@
...
+    "app.auction.riven.polarity": "ポラリチ",
```
Comment (Stream5710) [inline locales/ja.json:426]: `"app.auction.riven.polarity": "極性",`

File: locales/ja.json [hunk 9]
```diff
@@ -0,0 +1,932 @@
+ {
+    "app.disclaimer.de": "{warframeMarket} はファンサイト、そして {digitalExtremes} と付属ません。",
+    "app.general.title": "Warframe Market",
+    "app.general.description": "Warframe Marketはワーフレムセット、パーツ、武器、モド、レリック、設計図、キャプチュラーシーン、リベンとクバリッチおよびVaultとUnvaultのアイテムを売買する。",
+    "app.general.meta.tags": "warframe,market,buy,sell,trade,mod,blueprint,syndicate,relic,order,auction,wts,wtb,sales,worth,platinum,price,checking",
+
+    "app.form.field_required": "入力してください",
+    "app.form.can_not_be_null": "空白不可",
+    "app.form.invalid": "不当な入力",
```
Comment (Stream5710) [inline locales/ja.json:9]: `"app.form.invalid": "入力内容が無効です",`

File: locales/ja.json [hunk 10]
```diff
@@ -0,0 +1,932 @@
+ {
+    "app.disclaimer.de": "{warframeMarket} はファンサイト、そして {digitalExtremes} と付属ません。",
+    "app.general.title": "Warframe Market",
+    "app.general.description": "Warframe Marketはワーフレムセット、パーツ、武器、モド、レリック、設計図、キャプチュラーシーン、リベンとクバリッチおよびVaultとUnvaultのアイテムを売買する。",
+    "app.general.meta.tags": "warframe,market,buy,sell,trade,mod,blueprint,syndicate,relic,order,auction,wts,wtb,sales,worth,platinum,price,checking",
+
+    "app.form.field_required": "入力してください",
+    "app.form.can_not_be_null": "空白不可",
```
Comment (Stream5710) [inline locales/ja.json:8]: `"app.form.can_not_be_null": "空白にはできません",`

**Review by Stream5710 — COMMENTED** [review]
Hi, native Japanese speaker here. It looked like this was machine translated, so I have corrected some translations.
Sorry if it looked bad, this is my first time I have used GitHub's pull request review feature.

Some terms are displayed as is in JP client, Aside from specific item names, few examples are "(Riven) Mod", "Zaw", "Kitgun" and the names of polarity/Tenno schools.

**Comment by lonnstyle** [comment]
> Hi, native Japanese speaker here. It looked like this was machine translated, so I have corrected some translations.
> Sorry if it looked bad, this is my first time I have used GitHub's pull request review feature.
> 
> Some terms are displayed as is in JP client, Aside from specific item names, few examples are "(Riven) Mod", "Zaw", "Kitgun" and the names of polarity/Tenno schools.

Hi, sorry for the late reply. Here's wfm MOD, also the translator responsible for Chinese translations. Your corrections definitely help for improving our translation work quality. Would you mind to join our [Discord server](https://discord.gg/M7BHnPS), then dm me(lonnstyle#8677) or Head Moderator QueenHathena(Hathena#7777), so we can assign you the translator role and explains to you how does the translation work be done

**Pull request closed without merging**